src/Sentry.Serilog/SentrySinkExtensions.cs

```diff
@@ -159,132 +205,45 @@ public static class SentrySinkExtensions
-        }
-
-        if (!string.IsNullOrWhiteSpace(serverName))
-        {
-            sentrySerilogOptions.ServerName = serverName;
-        }
-
-        if (attachStackTrace.HasValue)
-        {
-            sentrySerilogOptions.AttachStacktrace = attachStackTrace.Value;
-        }
-
-        if (maxBreadcrumbs.HasValue)
-        {
-            sentrySerilogOptions.MaxBreadcrumbs = maxBreadcrumbs.Value;
-        }
-
-        if (sampleRate.HasValue)
-        {
-            sentrySerilogOptions.SampleRate = sampleRate;
-        }
-
-        if (!string.IsNullOrWhiteSpace(release))
-        {
-            sentrySerilogOptions.Release = release;
-        }
-
-        if (!string.IsNullOrWhiteSpace(environment))
-        {
-            sentrySerilogOptions.Environment = environment;
-        }
-
-        if (maxQueueItems.HasValue)
-        {
-            sentrySerilogOptions.MaxQueueItems = maxQueueItems.Value;
-        }
-
-        if (shutdownTimeout.HasValue)
-        {
-            sentrySerilogOptions.ShutdownTimeout = shutdownTimeout.Value;
-        }
-
-        if (decompressionMethods.HasValue)
-        {
-            sentrySerilogOptions.DecompressionMethods = decompressionMethods.Value;
-        }
-
-        if (requestBodyCompressionLevel.HasValue)
-        {
-            sentrySerilogOptions.RequestBodyCompressionLevel = requestBodyCompressionLevel.Value;
-        }
-
-        if (requestBodyCompressionBuffered.HasValue)
-        {
-            sentrySerilogOptions.RequestBodyCompressionBuffered = requestBodyCompressionBuffered.Value;
-        }
-
-        if (debug.HasValue)
-        {
-            sentrySerilogOptions.Debug = debug.Value;
-        }
-
-        if (diagnosticLevel.HasValue)
-        {
-            sentrySerilogOptions.DiagnosticLevel = diagnosticLevel.Value;
-        }
-
-        if (reportAssembliesMode.HasValue)
-        {
-            sentrySerilogOptions.ReportAssembliesMode = reportAssembliesMode.Value;
-        }
-
-        if (deduplicateMode.HasValue)
-        {
-            sentrySerilogOptions.DeduplicateMode = deduplicateMode.Value;
-        }
+        sentrySerilogOptions.Dsn = dsn ?? sentrySerilogOptions.Dsn;
+        sentrySerilogOptions.MinimumEventLevel = minimumEventLevel ?? sentrySerilogOptions.MinimumEventLevel;
+        sentrySerilogOptions.MinimumBreadcrumbLevel = minimumBreadcrumbLevel ?? sentrySerilogOptions.MinimumBreadcrumbLevel;
+        sentrySerilogOptions.FormatProvider = formatProvider ?? sentrySerilogOptions.FormatProvider;
+        sentrySerilogOptions.TextFormatter = textFormatter ?? sentrySerilogOptions.TextFormatter;
+        sentrySerilogOptions.SendDefaultPii = sendDefaultPii ?? sentrySerilogOptions.SendDefaultPii;
+        sentrySerilogOptions.IsEnvironmentUser = isEnvironmentUser ?? sentrySerilogOptions.IsEnvironmentUser;
+        sentrySerilogOptions.ServerName = NonNullOrWhiteSpaceOrDefault(serverName, sentrySerilogOptions.ServerName);
```

Comment:
Then need to introduce helper function in an attempt to reduce the amount of code here, when it was just 3 lines (if, then, assign)